src/data/markdown/translated-guides/en/03 Results output/100 End-of-test/150-custom-summary.md

```diff
@@ -41,7 +82,227 @@ The keys determine where k6 displays or saves the content:
 
 </CodeGroup>
 
-To get an idea how `data` would look in your specific test run, just add `return { 'raw-data.json': JSON.stringify(data)};` in your `handleSummary()` function and inspect the resulting `raw-data.json` file. Here's a very abridged example of how it might look:
+### Extract data properties
```

Comment:
Suggest making it explicit that this is a (dummy) example.

### Example: extract data properties

??